api/src/server/db/repository/access/account.repository.ts

```diff
@@ -21,21 +23,21 @@ export const createAccountRepository = (
       { email },
       { relations: ['characters'] }
```

Comment:
What do you think about moving relation types to entity itself and make it as a constant like 

export const CHARACTER_RELATION = 'characters'

Then in code we will have 

{ relations: [CHARACTER_RELATION] }

If field name be changed we would omit a problem with forgot to change name in relation

Reply:
something like that works. what do u think?
```ts
export const AccountRelations = {
  Characters: 'characters',
} as const;

@Entity({ name: 'access_accounts' })
export class Account {
  @PrimaryColumn('uuid')
  public id: string;

  @Column({ length: 320, unique: true })
  public email: string;

  @Column({ length: 320 })
  public passwordHash: string;

  @OneToMany(
    () => Character,
    character => character.account,
    {
      cascade: true,
    }
  )
  [AccountRelations.Characters]: Character[];
}

Reply:
Exacly!